File: python/list-libraries.py
#!/usr/bin/env python
"""
List libraries including owned, shared and group libraries.
"""

import urllib2

# replace with your token
token = '4da16bfbba92546df05f6ecfba68449be7a9f456'

url = 'https://seacloud.cc/api2/repos/'

req = urllib2.Request(url)
req.add_header('Authorization', 'Token ' + token)
response = urllib2.urlopen(req)
the_page = response.read()

print the_page

File: javascript/list-libraries.js
// replace with your own token
var token = '9dfa7a1f2dfa1c97550e55469bca9a68dcc7e03c'

var invocation = new XMLHttpRequest();
var url = 'https://seacloud.cc/api2/repos/';
function listLibraries() {
    if(invocation) {    
        invocation.open('GET', url, true);
        invocation.setRequestHeader("Authorization", "Token " + token);
        invocation.onreadystatechange = function() {
            if(invocation.readyState == 4 && invocation.status == 200) {
                console.log(invocation.responseText);
            }
        };

        invocation.send();
    }
}

listLibraries();
